src/browser/provider/built-in/phantomjs.js

```diff
@@ -0,0 +1,47 @@
+import { create as createPhantom } from 'phantom';
+
+export default {
+    phantom: null,
+
+    openedPages: {},
```

Comment:
Got it now. We now expect that provider can be used via alias without browser name, if `listAvailableBrowsers` returns null or empty array (BTW, I believe it should return `null` in that case, not empty array). Also, I don't like naming of this method then, maybe `listAvailableMutlibrowserOptions` or smthg, discuss with @VasilyStrelyaev